pandas/tests/indexes/multi/conftest.py

```diff
@@ -24,6 +24,18 @@ def idx():
     return mi
 
 
+@pytest.fixture
+def idx_multitype():
+    # a MultiIndex with several dtypes
+    first_axis = [1, 2, 3]
+    second_axis = list("abc")
+    third_axis = pd.date_range("20200101", periods=2, tz="UTC")
+    mi = MultiIndex.from_product(
```

Comment:
personal taste but I'd stick `first_axis`, `second_axis`, `third_axis` directly into the constructor